client/jetpack-cloud/sections/settings/advanced-credentials/index.tsx

```diff
@@ -79,7 +81,7 @@ const AdvancedCredentials: FunctionComponent = () => {
 			<SidebarNavigation />
 			<Card className="advanced-credentials__server-connection-status">
 				<div className="advanced-credentials__server-connection-status-content">
-					<h3>{ translate( 'Remote server connection credentials' ) }</h3>
+					<h3>{ translate( 'Remote server credentials' ) }</h3>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.